src/Nri/Ui/QuestionBox/V3.elm

```diff
@@ -270,7 +281,13 @@ viewBalloon config referencingId attributes =
     Balloon.view
         ([ Balloon.html
             (List.filterMap identity
-                [ Maybe.map (viewGuidance config referencingId) config.markdown
+                [ Just <|
+                    div [ css [ Css.displayFlex ] ]
```

Comment:
Hmm, but with the div, and with display flex, extra blank space gets added to the left side of the view. And I kinda think it's more than there should be? It's more than is on the right side of the question box for the character.

<img width="202" alt="Screen Shot 2023-01-31 at 9 25 40 AM" src="https://user-images.githubusercontent.com/8811312/215820250-1528a079-2cfe-4916-9e09-fcd720a1c354.png">
<img width="164" alt="Screen Shot 2023-01-31 at 9 25 45 AM" src="https://user-images.githubusercontent.com/8811312/215820254-08fc4d9b-141b-4f8d-b966-519e80bdeee4.png">

--

I think if the TTS stuff is positioned absolutely, it will be relative to the whole question box, so shouldn't matter whether there is a div here or not.

If I change the UiIcon.play icon positioning to absolute, and the left-button rendering back to:

```
(List.filterMap identity
                [ Just config.leftButton
                , Maybe.map (viewGuidance config referencingId) config.markdown
                , viewActions config.character config.actions
                ]
            )
```

it looks like this:

<img width="116" alt="Screen Shot 2023-01-31 at 9 30 18 AM" src="https://user-images.githubusercontent.com/8811312/215821405-c8c5abad-c480-45d0-92d1-f9bdfac05396.png">

Which is probably not enough whitespace on the left side! Maybe worth talking to Stacey about what the desired spacing is -- I could see wanting more left spacing iff TTS is present and I could see wanting more left spacing whether TTS is present or not.

Reply:
Again, though, I don't want to be a blocker on this. If Stacey says the current behavior is fine, that's fine with me.